services/event/db/schema.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose')
+const Joigoose = require('joigoose')(mongoose)
+const Joi = require('joi')
+
+const joiEventSchema = Joi.object({
+  dbType: Joi.string().required(),
```

Comment:
`dbType` is redundant, can be inferred from `dbURL`. It brings about complications later when a mysql url is provided with a db type of nosql.

Reply:
done